packages/integration_test/lib/_callback_io.dart

```diff
@@ -60,8 +61,8 @@ class IOCallbackManager implements CallbackManager {
     // comes up in the future. For example: `WebCallbackManager.cleanup`.
   }
 
-  // Whether the Flutter surface uses an Image.
-  bool _usesFlutterImage = false;
+  // Whether the Flutter surface uses an Image (Android only).
+  bool _usesFlutterImage = false || !Platform.isAndroid;
 
   @override
   Future<void> convertFlutterSurfaceToImage() async {
```

Comment:
it would need to noop if `!Platform.isAndroid`

Reply:
I meant to make it so it was never set to false on `!Platform.isAndroid`, let me clean this up.